src/plugins/intel_cpu/src/emitters/snippets/x64/kernel_executors/brgemm.hpp

```diff
@@ -99,10 +99,14 @@ struct brgemm_ref_kernel : public dnnl::impl::cpu::x64::brgemm_kernel_t {
     dnnl_status_t create_kernel() override {
         return dnnl_status_t::dnnl_success;
     }
-    [[nodiscard]] const dnnl::impl::cpu::x64::jit_generator* get_jit_generator() const override {
+    [[nodiscard]] const dnnl::impl::cpu::x64::jit_generator_t* get_jit_generator() const override {
         OV_CPU_JIT_EMITTER_THROW("get_jit_generator should not be called for reference kernel");
         return nullptr;
     }
+    const dnnl::impl::cpu::x64::brgemm_desc_t &get_brg() const override {
+        dnnl::impl::cpu::x64::brgemm_desc_t brg_desc;
+        return brg_desc;
+    }
```

Comment:
It's definitely a bad idea to return a reference to a local variable.

Reply:
> It's definitely a bad idea to return a reference to a local variable.

I see. That is because stock onednn base class imports a new pure member func. Above is just for compilation, not final.